### ISSUE TITLE
KSP 1.8 Never finishes loading with 7.13.0

### ISSUE THREAD

**Issue by jameslkey** [snjo/Firespitter#215]
KSP 1.8 runs through startup and makes it to the expansion loading where it hangs and continues to show loading screens.

**Comment by jameslkey** [comment]
New data... Firespitter Core loads but you can not click on any choices in the main menu.

**Comment by MikeLyno** [comment]
I'm having similar problems after 1.8 update. Hangs at verifying expansions. If I take out the Firesplitter folder, the game will load but then I can't load some bases/space stations that have some Firesplitter parts in them. So I'm essentially screwed the way things are right now since update.

**Comment by papasid26** [comment]
Got The same issue. in console it shows : "Verifying Expansion: Breaking ground SquadExpansion/Serenity". this issue is due to mechjeb and firespitter incompatibility. remove either one of those and it should work again.

**Comment by MikeLyno** [comment]
Nope. Just got the latest Mechjeb 2 today and it still hangs at the same place. Only way to get the game to load is deleting the Firesplitter folder but like I said, doing that is going to mess up some stuff you previously made. I sure hope the dudes at Firesplitter are reading these so than can investigate. They don't have working links to KSP Forums anymore to post about this.

**Comment by papasid26** [comment]
> 
> 
> New data... Firespitter Core loads but you can not click on any choices in the main menu.

yeah dude, tried with ckan and manual installs. firespitter core is the issue. mechjeb's developer sarbian also raised the issue of mod being compatible with 1.8.0 . lets wait for firespitter dev to fix it.

**Comment by enbock** [comment]
Take a look to my comment in #217 ;)

**Comment by MikeLyno** [comment]
Fixed in 7.14. Way to friggin go dudes! I can finally open the game now.  Wish you guys would join the KSP Forums again.
